app/views/users/show.html.erb

```diff
@@ -1 +1,25 @@
-<h1>Display the specific information of the user</h1>
\ No newline at end of file
+ <img src="<%= asset_path('im12.jpg') %>" alt="User Photo">
+<div class="specificUser">
+<h1>Name: <%= @user.name %></h1>
+<p><strong> Number of Posts:  <%= @user.posts.count %> </strong></p>
+</div>
+<div class="bio">
+<h2>Bio:</h2>
+<p><%= @user.bio %></p>
+</div>
+
+<% if @user.recent_posts.any? %>
+  <h2>Recent Posts:</h2>
+  <div class ="recPost">
+  <% @user.posts.each_with_index do |post, index| %>
+    <p><strong>Post#<%= index + 1 %>:</strong> <%= post.title %></p>
+    <p><strong>Post Detail:</strong> <%= post.text %></p>
+    <p><strong>Comments Count:</strong> <%= post.comments_counter %></p>
+    <p><strong>Likes Count:</strong> <%= post.likes_counter %></p>
+    <hr>
```

Comment:
- [ ] According to the project requirement, you are meant to use the `recent_posts` method to return the 3 most recent posts and display them, currently, you are checking if they exist, but then go ahead to loop and display all the users post and that's not correct. You are meant to use the method to return the 3 posts and then loop and display them. :+1: :+1: 
```erb
<% @user.recent_posts do |post| %>
```

- [ ] Just as I pointed out above about following the given wireframe, you also need to update this.
- - Your current Design:
![Screenshot from 2023-11-24 08-51-48](https://github.com/faisalakb/Rails-Blog-App/assets/107632780/98641530-0d45-4e17-bbb8-32dc90f31a19)

- - Wireframe:
![Screenshot from 2023-11-24 08-51-59](https://github.com/faisalakb/Rails-Blog-App/assets/107632780/1195ee2a-a001-4f57-811a-c7f20e529fb9)

- [ ] Also this `<img src="<%= asset_path('im12.jpg') %>" alt="User Photo">` is not totally correct. The user's photo is meant to be from the user's photo link in the database and not a hardcoded image file. :+1: :100: